package/yast2-alternatives.spec

```diff
@@ -15,23 +15,22 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+
 Name:           yast2-alternatives
 Version:        4.2.0
```

Comment:
Please bump the version to 4.2.1 so the change is automatically submitted to Factory by Jenkins.